app/screens/wallet/Backup.js

```diff
@@ -0,0 +1,187 @@
+// @flow
+import React, { Component } from 'react';
+import styled, { css } from 'styled-components';
+import { smColors } from '/vars';
+import { shieldIconGreenOne, shieldIconOrangeTwo } from '/assets/images';
+
+const Actionable = css`
+  &:hover {
+    opacity: 0.8;
+  }
+  &:active {
+    opacity: 0.6;
+  }
+`;
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 50px;
+`;
+
+const Header = styled.span`
+  font-size: 31px;
+  font-weight: bold;
+  line-height: 42px;
+  color: ${smColors.lighterBlack};
+  margin-bottom: 20px;
+`;
+
+const BaseText = styled.span`
+  font-size: 16px;
+  font-weight: normal;
+  line-height: 22px;
+  color: ${smColors.lighterBlack};
+`;
+
+const BoldText = styled(BaseText)`
+  font-weight: bold;
+`;
+
+const GrayText = styled(BaseText)`
+  color: ${smColors.darkGray};
+`;
+
+const Separator = styled.div`
+  border-bottom: 1px solid ${smColors.borderGray};
+  margin: 12px 0;
+`;
+
+const BackupBoxesWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 12px 0;
+`;
+
+// $FlowStyledIssue
+const ActionLink = styled(BaseText)`
+  user-select: none;
+  color: ${({ theme }) => (theme === 'orange' ? smColors.orange : smColors.green)};
+  cursor: pointer;
+  ${Actionable}
+`;
+
+// $FlowStyledIssue
+const BackupBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  ${({ borderColor }) =>
```

Comment:
border color is provided in any case - simplify

Reply:
ok